storybook/src/components/FieldSet/FieldSet.stories.tsx

```diff
@@ -38,9 +40,16 @@ type Story = StoryObj<typeof meta>
 
 export const Default: Story = {
   render: (args) => (
-    <FieldSet invalid={args.invalid} legend={args.legend}>
+    <FieldSet
+      invalid={args.invalid}
+      legend={args.legend}
+      required={args.required}
+      notRequiredLabel={args.notRequiredLabel}
+    >
       <Field className="ams-mb--sm">
-        <Label htmlFor="input-a1">Voornaam</Label>
+        <Label htmlFor="input-a1" required={args.required} notRequiredLabel="">
```

Comment:
`required` and `notRequiredLabel` shouldn't be defined here, right? You set the label on the Field Set legend?